exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/fn/JsonReader.java

```diff
@@ -402,7 +402,8 @@ private void writeDataAllText(MapWriter map, FieldSelection selection, boolean m
         handleString(parser, map, fieldName);
         break;
       case VALUE_NULL:
-        // do nothing as we don't have a type.
```

Comment:
Won't this cause both of these files to fail to read?

FILE1
{a: 4, b:null}
{a:6, b: [1,2,3]}

FILE2
{a:4, b:null}
{a:6, b: {x:1, y:2}}